core_test/lwt/dune

```diff
@@ -0,0 +1,22 @@
+(* -*- tuareg -*- *)
+
+let coverage_rewriter =
+  let bisect = Jbuild_plugin.V1.run_and_read_lines "echo ${BISECT_ENABLE}"
+    |> String.concat ""
+  in
+  if bisect = "YES" then "bisect_ppx -conditional" else ""
+
+let () = Printf.ksprintf Jbuild_plugin.V1.send {|
+(executables
+  (names
+    link_test_main
```

Comment:
Unfortunately dune won't build these if they don't have a public name. Once we move to dune 1.5 we can do a `dune build @check` to test that everything still builds and links, in the meantime this is fine.

Reply:
?? But I can run the tests ok (the Lwt test calls client_main and server_main):
```
$ make test
dune runtest --no-buffer --profile=release
Done: 505/507 (jobs: 1)Checking the switch can start late
Finished 584 RPCs in 5.00
Performance test of Unix to Unix
Finished 573 RPCs in 5.01
Performance test of Lwt to Lwt
Finished 529 RPCs in 5.00
Performance test of Async to Lwt
Finished 559 RPCs in 5.00
Performance test of Async to Async
Finished 777 RPCs in 5.00
```

Reply:
Ah they are referred to from https://github.com/xapi-project/message-switch/blob/4ed7162896e4bd3c3957105d37a767c68ddc2897/core_test/jbuild#L28-L37, ignore my previous comment.